src/test/java/CreatePlayList.java

```diff
@@ -0,0 +1,136 @@
+import Auto.pages.AutoHomePage;
+import Auto.pages.AutoLoginPage;
+import Auto.pages.AutoPlaylistPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CreatePlayList extends BaseTest1{
+
+    By enterNameList = By.cssSelector("[type ='text]");
+    String playListName = "my list";
+    String firstQ = "\"";
+    String secondQ = ".";
+    String lastQ = "\"";
+    String name= "\""+ playListName+"."+"\"";
+    String tenChars="TenthTenth";
+    String two="two";
+    String elevenChars="elevenChars";
+
+
+    @Test(enabled = true, priority = 0)
+    public void createNewList () throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Reply:
I deleted all throws InterruptedException for all methods in class PlayListTests